docs/DeveloperGuide.md

```diff
@@ -601,6 +601,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *` | power user | access my command history | efficiently repeat similar commands |
 | `* *` | power user | access command history from previous sessions | easily reuse commands from previous sessions |
 | `* *` | power user | create aliases for longer commands | avoid typing the same command multiple times |
+| `* *` | power user | delete aliases that are no longer needed | |
```

Comment:
Maybe provide a justification for this user story? It could be asked "what's the benefit of deleting aliases? Just don't use the ones you need anymore."